middleware/rewrite/rewrite_test.go

```diff
@@ -20,9 +20,9 @@ func TestRewrite(t *testing.T) {
 	rw := Rewrite{
 		Next: middleware.HandlerFunc(msgPrinter),
 		Rules: []Rule{
-			NewSimpleRule("from.nl.", "to.nl."),
-			NewSimpleRule("CH", "IN"),
-			NewSimpleRule("ANY", "HINFO"),
+			Fields["qname"].New("from.nl.", "to.nl."),
```

Comment:
We should try to use `example.*` zone in these tests (yes, there is a of `miek.nl` and `.nl` out there still)